src/Control/Category/Tensor.hs

```diff
@@ -58,26 +49,36 @@ instance GBifunctor (Star Maybe) (Star Maybe) (Star Maybe) These where
       That c -> That <$> g c
       These a c -> liftA2 These (f a) (g c)
 
-grmap :: GBifunctor cat1 cat2 r t => c `cat2` d -> t a c `r` t a d
+grmap :: GBifunctor cat1 cat2 cat3 t => cat2 c d -> cat3 (a `t` c) (a `t` d)
 grmap = gbimap id
 
-glmap :: GBifunctor cat1 cat2 r t => a `cat1` b -> t a c `r` t b c
+glmap :: GBifunctor cat1 cat2 cat3 t => cat1 a b -> cat3 (a `t` c) (b `t` c)
 glmap = flip gbimap id
 
-data Iso cat a b = Iso { fwd :: a `cat` b, bwd :: b `cat` a }
-
+data Iso cat a b = Iso { fwd :: cat a b, bwd :: cat b a }
+
+-- | An Associative Bifunctor is one whose type operator 't' is
+-- associative.
```

Comment:
Somewhere in this haddock the word "isomorphism" should show up.

Reply:
Good call. I added a haddock to `Iso`